jupyterhub/jupyterhub_config.py

```diff
@@ -1,10 +1,42 @@
+import os
+
+from tornado import gen
+from jupyterhub.auth import Authenticator
+
 # Configuration file for jupyterhub.
 from jupyter_client.localinterfaces import public_ips
 
 c = get_config()  # noqa
 
 c.JupyterHub.ssl_key = 'test.key'
 c.JupyterHub.ssl_cert = 'test.crt'
-c.JupyterHub.spawner_class = 'remoteappmanager.spawner.Spawner'
 
 c.JupyterHub.hub_ip = public_ips()[0]
+
```

Comment:
You can move it... It's ok if it's part of it. It's a good utility and moving it keeps the config clean.